server/superdesk/publish/publish_content.py

```diff
@@ -63,8 +63,16 @@ def publish(subscriber, destination):
 def transmit_items(queue_items, subscriber, destination):
     failed_items = []
 
+    output_channels = superdesk.get_resource_service('output_channels').get(req=None, lookup={})
```

Comment:
Just a minor thing: Since we know the output_channel_id at this point we can use the lookup to get that one only: 
```
lookup = {'_id': 'output_channel_id'}
```

Or we can do get all of them in previous function and feed the output channel to each thread starts with .apply_sync. What do you think?